### ISSUE TITLE
Content Page editMode Javascript error on NG Iframe Component

### ISSUE THREAD

**Issue by oswaldogallango** [dotCMS/core#9300]
When you are editing a Content Page or HTML Page  in edit editMode using the NG Iframe Component, after you add or edit a content or container and return to the editMode view. The the javascript stop working throwing an exception like  Uncaught TypeError: Cannot read property 'frameMain' of undefined  and that you don't have permission to access href property or if you try to click one of the options in the left menu side try to open a new that window with an exception in the javascript

This issue is related to https://github.com/dotCMS/core/issues/9168

**Comment by oswaldogallango** [comment]
In this iteration were fixed the following issues generated after you enter in edit mode on any Page content, add or edit a content or edit page/template and return to the edit mode page:

1) the live, previews and edit mode tabs didn't work, throwing a javascript exception
2) If you open a container controller in edit mode the close option didn't work, throwing a javascript error
3) in some browser the target detailFrame wasn't working because the attribute name was not present in the iframe

The remaining problems are described in this issue https://github.com/dotCMS/core/issues/9811

**Comment by oswaldogallango** [comment]
Pulls https://github.com/dotCMS/core-web/pull/193 and https://github.com/dotCMS/core/pull/9812

**Comment by bryanboza** [comment]
Some issues pending on the new card
